### PR TITLE
Refactoring css and html files

### PR DESCRIPTION
**Pull request by rohan09-raj**
Refactored and rewritten all the CSS for main page
Please check to deploy preview for the visual view.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,202 +22,221 @@
...
+              />
+            </a>
+          </div>
+        </span>
+        <img
+          class="welcome-image"
+          src="../img/re-designed-images/welcome-image.png"
+          alt="Welcome_Image"
+        />
       </div>
```
Comment (Rucha1499) [inline index.html:222]: image
Can we increase the space between them for mobile screens? Its looking a little crowded.

**Review by Rucha1499 — CHANGES_REQUESTED** [review]
Thanks for refactoring the code and fixing the design! @rohan09-raj 💯 

Just requesting some minor changes. Please have a look at them.

File: index.html [hunk 2]
```diff
@@ -24,202 +22,221 @@
...
-            class="handles-logos"
-          />
-        </a>
+            <span>
+              <img
+                src="../img/re-designed-images/separator-icon.png"
+                alt="Separator_Icon"
+                class="separator"
+              />
+            </span>
```
[the remark this answers is not in the document]
Comment (rohan09-raj) [reply index.html:158]: Yess definitely will change that.

File: index.html (hunk 1 above)
Comment (rohan09-raj) [reply index.html:222]: Yes sure

**Review by Ishikabaid — CHANGES_REQUESTED** [review]
Screenshot (6)
Can we move this colon with the above text? It's looking very weird here.

**Comment by rohan09-raj** [comment]
@Ishikabaid Can you please mention the dimensions ??

**Comment by Ishikabaid** [comment]
> @Ishikabaid Can you please mention the dimensions ?? Because in my local at all the breakpoints it is fine.

I checked it for Moto G4

**Comment by rohan09-raj** [comment]
> > @Ishikabaid Can you please mention the dimensions ?? Because in my local at all the breakpoints it is fine.
> 
> I checked it for Moto G4

Resolved. Please check now

**Comment by Ishikabaid** [comment]
> > > @Ishikabaid Can you please mention the dimensions ?? Because in my local at all the breakpoints it is fine.
> > 
> > 
> > I checked it for Moto G4
> 
> Resolved. Please check now

Thanks for the changes @rohan09-raj . Looks good to me🎉

**Review by Ishikabaid — APPROVED** [review]

**Review by Rucha1499 — APPROVED** [review]
Thank you for fixing this! @rohan09-raj 🎉

**Comment by rohan09-raj** [comment]
> Thank you for fixing this! @rohan09-raj 🎉

You're welcome 😃

**Pull request merged**